### PR TITLE
fix(Scripts/Spells): Remove some magic numbers

### PR DESCRIPTION
**Pull request by Kitzunu**
## Changes Proposed:
-  no changes
-  

## Issues Addressed:

- Closes 

## Tests Performed:

- 
- 

## How to Test the Changes:

1.
2.
3.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: src/server/scripts/Spells/spell_item.cpp [hunk 1]
```diff
@@ -159,6 +169,12 @@ class spell_item_runescroll_of_fortitude : public SpellScript
     }
 };
 
+enum BrannsCommunicator
+{
+    NPC_BRANN_BRONZEBEARD = 29579,
+    SPELL_CONTACT_BRANN   = 55038,
+}
```
Comment (Kitzunu) [inline src/server/scripts/Spells/spell_item.cpp:176]:
```suggestion
};
```

File: src/server/scripts/Spells/spell_item.cpp [hunk 2]
```diff
@@ -136,6 +141,11 @@ class spell_item_mind_amplify_dish : public SpellScript
     }
 };
 
+enum RunescrollOfFortitude
+{
+    SPELL_FORTITUDE = 72590,
+}
```
Comment (Kitzunu) [inline src/server/scripts/Spells/spell_item.cpp:147]:
```suggestion
};
```

File: src/server/scripts/Spells/spell_item.cpp [hunk 3]
```diff
@@ -31,6 +31,11 @@
 #include "SpellScript.h"
 #include "WorldSession.h"
 
+enum MassiveSeaforiumCharge
+{
+    ITEM_MASSIVE_SEAFORIUM_CHARGE = 39213,
+}
```
Comment (Kitzunu) [inline src/server/scripts/Spells/spell_item.cpp:37]:
```suggestion
};
```

**Review by acidmanifesto — APPROVED** [review]

**Comment by acidmanifesto** [comment]
Thank you for hte PR

**Pull request merged**